flink-connectors/flink-connector-kafka-0.9/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer09.java

```diff
@@ -311,4 +323,21 @@ private static void setDeserializer(Properties props) {
 		props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, deSerName);
 		props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, deSerName);
 	}
+
+
+	// ------------------------------------------------------------------------
```

Comment:
Unless there is precedence let's remove this comment - it is pretty obvious what these methods are there for :)

Reply:
Sounds good, removed :)